package.json

```diff
@@ -1,14 +1,18 @@
 {
   "name": "@exercism/codemirror-lang-wren",
-  "description": "Wren language support for the CodeMirror code editor",
+  "description": "Wren Lezer parser and language support for CodeMirror",
   "author": "Josh Goebel <me@joshgoebel.com>",
-  "version": "0.6.0",
+  "version": "0.7.0",
   "license": "MIT",
   "repository": {
     "type": "git",
     "url": "https://github.com/exercism/codemirror-lang-wren"
   },
-  "main": "dist/index.cjs",
+  "imports": {
+    "#parser": "./src/lezer/parser.js"
+  },
+  "type": "module",
+  "main": "dist/index.es.js",
```

Comment:
You probably want to keep the `.cjs` for `"main"` _if_ you want to support legacy nodes. If you don't want to support legacy nodes, you don't need `"main"` at all.

> If both "exports" and "main" are defined, the "exports" field takes precedence over "main". "exports" are not specific to ES modules or CommonJS; "main" is overridden by "exports" if it exists. As such "main" cannot be used as a fallback for CommonJS but it can be used as a fallback for legacy versions of Node.js that do not support the "exports" field.

They still recommend it tho, but without explanation:

> To set the main entry point for a package, it is advisable to define both "exports" and "main" in the package’s package.json file:
>
> ```json
> {
>   "main": "./main.js",
>   "exports": "./main.js"
> }
> ```

Reply:
I pointed main to cjs.  Anything else?